src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -212,7 +222,22 @@ public async Task<IActionResult> BulkEdit(WorkflowTypeIndexOptions options, IEnu
             return RedirectToAction(nameof(Index));
         }
 
-        public async Task<IActionResult> EditProperties(long? id, string returnUrl = null)
+        public async Task<IActionResult> Export(int id)
+        {
+            if (!await _authorizationService.AuthorizeAsync(User, Permissions.ManageWorkflows))
+            {
+                return Forbid();
+            }
+
+            var resultFileName = await ExportWorkflows(id);
+
+            return new PhysicalFileResult(resultFileName, "application/zip")
+            {
+                FileDownloadName = "WorkflowType.zip"
+            };
```

Comment:
I'd put this into `ExportWorkflows` too (and use plural/singular for the file name there).